.github/workflows/rust.yml

```diff
@@ -202,8 +202,15 @@ jobs:
       - name: Checkout source code
         uses: actions/checkout@v2
 
-      - name: Remove default members
-        run: sed '/default-members/d' -i ${{ matrix.directory }}/Cargo.toml
+      # Store `Cargo.toml` before modifying it to not affect the diff.
+      - name: Remove and store default members
+        id: default_members
+        run: |
+          cargo_toml=$(< ${{ matrix.directory }}/Cargo.toml)
+          echo "ORIGINAL_CARGO_TOML<<EOF" >> $GITHUB_ENV
+          echo "$cargo_toml" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
```

Comment:
Instead of dancing around with the environment variable, can't we simple do this?
```bash
cp "${{ matrix.directory }}/Cargo.toml" "${{ matrix.directory }}/Cargo.toml.orig"
```
and later
```bash
mv "${{ matrix.directory }}/Cargo.toml.orig" "${{ matrix.directory }}/Cargo.toml"
```

Reply:
Yeah the env var started simple and ended up sooo complicated...

Reply:
I actually liked the idea of the environment variable, I wasn't aware of `$GITHUB_ENV`, this could be useful in other places as well. Also using an output is probably the cleanest way but very verbose and hard to follow. This solution is probably the simplest **and most robust** one.